traffic_monitor/towrap/towrap.go

```diff
@@ -229,16 +244,36 @@ func (s *TrafficOpsSessionThreadsafe) CRConfigValid(crc *tc.CRConfig, cdn string
 
 // CRConfigRaw returns the CRConfig from the Traffic Ops. This is safe for multiple goroutines.
 func (s TrafficOpsSessionThreadsafe) CRConfigRaw(cdn string) ([]byte, error) {
+
 	ss := s.get()
+
+	var b []byte
+	var err error
+	var remoteAddr string
+	var reqInf client.ReqInf
+
 	if ss == nil {
 		return nil, ErrNilSession
+	} else {
```

Comment:
Nitpick: since the above `if` returns, we don't need an `else` here and can reduce nesting. That also lets us avoid the `var` pre-declarations and `b, reqInf, err := ss.GetCRConfig(cdn)`